### PR TITLE
Towards genesis ceremony test

### PR DESCRIPTION
**Pull request by adaszko**
First batch of https://rchain.atlassian.net/browse/RCHAIN-2172

### CODE REVIEW

**Comment by adaszko** [comment]
bors try

**Comment by adaszko** [comment]
bors try

**Comment by adaszko** [comment]
bors try

File: integration-tests/test/conftest.py [hunk 1]
```diff
@@ -88,17 +88,21 @@ def docker_client() -> Generator[DockerClient, None, None]:
         docker_client.networks.prune()
 
 
+@pytest.yield_fixture(scope='session')
+def random_generator(command_line_options: CommandLineOptions) -> Generator[Random, None, None]:
+    random_seed = time.time() if command_line_options.random_seed is None else command_line_options.random_seed
+    logging.critical("Using tests random number generator seed: %d", random_seed)
```
Comment (woky) [inline integration-tests/test/conftest.py]: <bikeshed>s/critical/info/ ?</bikeshed>

Comment (adaszko) [reply integration-tests/test/conftest.py]: The log level in CI is at ERROR.  I need this message to be *always* printed.

**Pull request merged**